app/src/org/commcare/android/util/SessionUnavailableException.java

```diff
@@ -1,19 +1,14 @@
-/**
- * 
- */
 package org.commcare.android.util;
 
 /**
- * @author ctsims
+ * Signals the expiration of the session created on login that controls
+ * liveness of user database and key pool.
  *
+ * @author ctsims
  */
-public class SessionUnavailableException extends RuntimeException {
-
-    public SessionUnavailableException() {
-        super();
-    }
-
-    public SessionUnavailableException(String message) {
-        super(message);
-    }
+public class SessionUnavailableException extends Exception {
```

Comment:
Make this extend a Exception (which forces checking) instead of RuntimeException.